### ISSUE TITLE
[HelpHub] Feedback on Cookies

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#756]
## Issue Description
Originally reported by https://profiles.wordpress.org/stefanocianchi/

>Cookies are blocked on FireFox. Just write how to remove the blockage. Why do I need to read hundreds of lines and, at the end, I do not know how to fix the problem?

## URL of the Page with the Issue

https://wordpress.org/documentation/article/cookies/

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.
